pkg/bootstrap/bootstrap_config.go

```diff
@@ -286,6 +286,10 @@ func WriteBootstrap(config *meshconfig.ProxyConfig, node string, epoch int, pilo
 
 	// Pass unmodified config.DiscoveryAddress for Google gRPC Envoy client target_uri parameter
 	opts["discovery_address"] = config.DiscoveryAddress
+	// Check if nodeIP carries IPv4 or IPv6 and set up proxy accordingly
+	if isIPv6Addr(nodeIPs[0]) {
+		opts["ipv6_proxy"] = "true"
+	}
```

Comment:
this is also a hack.. nodes can have mixed mode IPs. Kubernetes can't but VMs can. So, this is not the right way of enabling ipv6_proxy..

Reply:
I agree, we should iterate trough all addresses. If at least one IP4 is found - we can probably use the ip4 style. 
If all are IP6 - I think we need to use ip6 style.

Reply:
Ok. I will loop through all IPs. My concern is if we have valid ipv6 addresses and only 1 ipv4 127.0.0.1, then falling back to ipv4 does not make sense. Would you agree?